config.example.yaml

```diff
@@ -60,10 +60,6 @@ services:
                     retry_delay: 250
                     retry_limit: 10
                     show_sql: false
-                  citoid:
```

Comment:
Why is this being removed?

Reply:
Because the example.yaml config don't specify citoid or mathoid modules, so the setting is useless.